### PR TITLE
	Added tests for enable or disable RC functionality from HMI

### PR DESCRIPTION
**Pull request by aderiabin**

### CODE REVIEW

File: test_scripts/RC/OnRemoteControlSettings/009_Allowed_true_accessMode_AUTO_ALLOW.lua [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+runner.Step("Enable RC from HMI with AUTO_ALLOW access mode", commonRC.defineRAMode, { true, "AUTO_ALLOW"})
+for _, mod in pairs(modules) do
+  for _, rpc in pairs(rcRpcs) do
+    runner.Step("Activate App1", commonRC.activate_app)
+    runner.Step("Check module " .. mod .." App1 " .. rpc .. " allowed", commonRC.rpcAllowed, { mod, 1, rpc })
+    runner.Step("Activate App2", commonRC.activate_app, { 2 })
+    runner.Step("Check module " .. mod .." App2 " .. rpc .. " allowed", commonRC.rpcAllowed, { mod, 2, rpc })
+  end
+end
+
```
Comment (Itileda) [inline test_scripts/RC/OnRemoteControlSettings/009_Allowed_true_accessMode_AUTO_ALLOW.lua]: Sequence: disable-enable-check_if_allowed
better to replace with 
sequence: disable-check_if_disallowed-enable-check_if_allowed

File: test_scripts/RC/OnRemoteControlSettings/013_sequence_switches_of_accessMode.lua [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+-- [SDL_RC] Resource allocation based on access mode
+--
+-- Description:
+-- In case:
+-- RC_functionality is disabled on HMI and HMI sends notification OnRemoteControlSettings (allowed:true, <any_accessMode>)
+--
+-- SDL must:
+-- 1) store RC state allowed:true and received from HMI internally
+-- 2) allow RC functionality for applications with REMOTE_CONTROL appHMIType
+---------------------------------------------------------------------------------------------------
```
Comment (Itileda) [inline test_scripts/RC/OnRemoteControlSettings/013_sequence_switches_of_accessMode.lua]: Unclear description
what is the difference with 009-012?

Comment (aderiabin) [inline test_scripts/RC/OnRemoteControlSettings/013_sequence_switches_of_accessMode.lua]: @Itileda Difference in goal of test. Description was updated in

File: test_scripts/RC/OnRemoteControlSettings/009_Allowed_true_accessMode_AUTO_ALLOW.lua (hunk 1 above)
Comment (aderiabin) [inline test_scripts/RC/OnRemoteControlSettings/009_Allowed_true_accessMode_AUTO_ALLOW.lua]: @Itileda 
sequence: disable-check_if_disallowed is covered in test_scripts/RC/OnRemoteControlSettings/008_Allowed_false.lua
This check is redundant for current test.

Comment (Itileda) [inline test_scripts/RC/OnRemoteControlSettings/009_Allowed_true_accessMode_AUTO_ALLOW.lua]: @aderiabin, ok accepted.

**Pull request merged**